relay-server/src/actors/envelopes.rs

```diff
@@ -2002,11 +1997,23 @@ impl EnvelopeManager {
         }
 
         relay_log::trace!("sending envelope to sentry endpoint");
+
+        let http_encoding = self.config.http_encoding();
+        let (envelope_body, envelope_meta) = match Self::encode_envelope(envelope, http_encoding) {
+            Ok(v) => v,
+            Err(e) => {
+                return Box::new(future::err(SendEnvelopeError::SendFailed(
```

Comment:
i'm not particularly happy about the type of this error cause it's all backwards. i was thinking of making a new variant of SendEnvelopeError (two actually, a BodyEncodingError(std::io::Error) and an EnvelopeBuildError(EnvelopeError)), but then i'd have to carry them into ProcessingError too, and i'm not 100% sure it's worth it? idk

Reply:
I'd do exactly that, it's fine to have so many variants

Reply:
ok will do